### ISSUE TITLE
Better Tool Options

### ISSUE THREAD

**Issue by mamasch19** [adil192/saber#42]
I would love to see an option to change the brush size or to manually pick a color and save it as a favorite.

**Comment by comcloudway** [comment]
I think one should also be able to change the `highlighter` brush size

**Comment by adil192** [comment]
> I think one should also be able to change the `highlighter` brush size

Yeah I realised after I ticked it off that I hadn't done it for the highlighter :p -- I've added that to the roadmap

**Comment by adil192** [comment]
I've yet to implement saving the brush settings/size, also in the roadmap :)
